styles/style.css

```diff
@@ -81,20 +89,146 @@ textarea {
     appearance: none;
     border-radius: 0;
 }
+
 .btn {
     display: flex;
     justify-content: center;
-	align-items: center;
+    align-items: center;
     padding: 8px;
     width: 100%;
     max-width: 94px;
     border-radius: 2px;
     color: var(--color-white);
     font: var(--font-button-default);
 }
+
 .btn.btn--primary {
     padding: 16px;
     max-width: 163px;
     background: var(--color-accent);
     font: var(--font-button-small);
 }
+
+ul {
+    list-style-type: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+.container {
+    margin: 0 auto;
+    width: 100%;
+    max-width: 1177px;
+}
+
+.site-header .header-top {
+    padding: 14px 0;
+    background-color: #7E33E0;
+}
+.site-header .header-top a {
+    color: var(--color-white);
+    line-height: 18.75px;
+    font-size: 16px;
+    font-family: Arial, "Josefin Sans", sans-serif;
+}
+.site-header .header-top .container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 48px;
+
+}
+.site-header .header-top .container .header-left {
+    display: flex;
+    align-items: center;
+    gap: 48px;
+}
+.site-header .header-top .container .connection {
+    display: flex;
+    align-items: center;
+    gap: 10px;
+    max-height: 16px;
+
+}
+
+.site-header .header-top .container .header-right {
+    display: flex;
+    align-items: center;
+    gap: 32px;
+}
+.site-header .header-top .container .header-right a {
+    display: flex;
+    align-items: center;
+    gap: 2px;
+}
+.site-header .header-top .container .header-right button {
+    border: unset;
+    background: transparent;
+}
+.header-bottom {
+    padding: 19px 0;
+}
+
+.header-bottom .container .header-bottom-row {
+    display: flex;
+    align-items: center;
+    background-color: var(--color-white);
+}
+.header-bottom .container .header-bottom-row .header-logo {
+  padding-right: 88px;
+}
+.header-bottom .container .header-bottom-row .header-nav {
+    display: flex;
+    justify-content: space-between;
+}
+
+.header-bottom .container .header-bottom-row .header-nav ul {
+    display: flex;
+    column-gap: 35px;
+}
+
+.header-bottom .container .header-bottom-row .header-nav ul li a {
+    color: #0D0E43;
+    line-height: 20px;
+    font-weight: 400;
+    font-size: 16px;
+    font-family: Arial, "Lato", sans-serif;
+    transition: 0.25s;
+    cursor: pointer;
+    
+}
+.header-bottom .container .header-bottom-row .header-nav ul li a:hover {
+    color: var(--color-accent);
+}
+.header-bottom .container .header-bottom-row .header-nav ul li .arrow {
+display: flex;
+align-items: center;
+gap: 2px;
+height: 100%;
+}
+.header-bottom .container .header-bottom-row .form {
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+    height: 40px;
+    border-width: 2px;
+    border-color: #E7E6EF;
+}
+.header-bottom .container .header-bottom-row .form input {
```

Comment:
with of the input doesn't match the layout in figma
<img width="303" alt="image" src="https://github.com/user-attachments/assets/ef357d75-5f95-4c23-b05e-e2d3714e2039">

Reply:
corrected

Reply:
No, it's the same. Before you commit, double-check what you're doing.